pontoon/base/static/js/fluent_interface.js

```diff
@@ -1,15 +1,39 @@
 /* global FluentSyntax */
 var fluentParser = new FluentSyntax.FluentParser({ withSpans: false });
+var fluentParserOriginal = new FluentSyntax.FluentParser({ withSpans: false });
```

Comment:
Rather than instantiating the parser twice and overwriting `parseEntry` in one of the instances, why not write a custom `parseEntry` function and use it throughout the file when needed? It should save some memory and also make this work-around a bit nicer. The custom logic belongs in Pontoon, rather than in `FluentParser` after all.

Reply:
I was thinking about that initially, but then I changed my mind, because it's a temporary workaround and eventually we'd need to replace `parseEntry()` with `fluentParser.parseEntry()` again. Do you think I should still do it?

Reply:
We'll need to change all the callsites which use `useDefaultParser = false` anyways :) This might be a personal preference, but if possible, I'd go for wrappers rather than monkey patches.

Reply:
Or, in other words:

![image](https://user-images.githubusercontent.com/265818/43830604-f7f6d83e-9b01-11e8-80a3-3d9c52dabde6.png)

Reply:
OK, I'll do that in a bit.